src/app/features/animals/components/animals-list/animals-list.component.html

```diff
@@ -1,4 +1,60 @@
-<h1>Animals</h1>
+<div class="row">
+  <div class="col">
+    <h1>Animals</h1>
+  </div>
+  <div class="col text-end">
+    <button class="btn btn-success" routerLink="/animals/add">Add new</button>
+  </div>
+</div>
 
-<!-- TODO: Add table with animals -->
-<p>Coming soon...</p>
+<ul class="nav nav-tabs mb-4">
+  <li class="nav-item">
+    <a class="nav-link" (click)="changeActiveTab('table')" [class.active]="activeTab === 'table'">Table</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" (click)="changeActiveTab('cards')" [class.active]="activeTab === 'cards'">Cards</a>
+  </li>
+</ul>
+
+<div>
+  <ng-container *ngIf="activeTab === 'table'">
```

Comment:
Po implementácii `routerLink` z predchádzajúceho komentára bude lepšie preniesť obsah v `ng-container` do samostatných komponentov a využiť `router-outlet`.

```suggestion
  <router-outlet></router-outlet>
```